lib/evss/disability_compensation_form/data_translation.rb

```diff
@@ -19,6 +19,7 @@ def initialize(user, form_content, _)
       #
       def translate
         form['claimantCertification'] = true
+        form['autoCestPDFGenerationDisabled'] ||= false
```

Comment:
Not your fault, but the double negative (...disabled is false, which means enabled) was a little hard to understand but makes sense in the context of what's at the other end of the API. You want it to NOT DISABLE autogeneration by default which is the current behavior.

Reply:
Yes, because the generation happens down stream there really isn't a way that I can think of to really test this on the application itself. We don't even get a response back from EVSS on when this occurs so difficult to even use a mock in the test when returning a response etc. The `||=` is intended to trigger only when form['autoCestPDFGenerationDisabled'] is nil or not included which should never happen with the json api validation but good to catch none the less.